src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py

```diff
@@ -1074,6 +1244,139 @@ def test_import_zaaktype_invalid_eigenschap(self, *mocks):
         self.assertEqual(ZaakType.objects.count(), 0)
         self.assertEqual(Eigenschap.objects.count(), 0)
 
+    def test_import_zaaktype_informatieobjectype_overlapping(self, *mocks):
+        catalogus = CatalogusFactory.create(rsin="000000000", domein="TEST")
+        zaaktype = ZaakTypeFactory.create(
+            catalogus=catalogus,
+            vertrouwelijkheidaanduiding="openbaar",
+            zaaktype_omschrijving="bla",
+            datum_begin_geldigheid="2023-01-01",
+        )
+
+        informatieobjecttype = InformatieObjectTypeFactory.create(
+            catalogus=catalogus,
+            vertrouwelijkheidaanduiding="openbaar",
+            omschrijving="Alpha",
+            zaaktypen__zaaktype=zaaktype,
+            datum_begin_geldigheid="2023-01-01",
+        )
+
+        # create zip
+        url = reverse("admin:catalogi_zaaktype_change", args=(zaaktype.pk,))
+        response = self.app.get(url)
+        form = response.forms["zaaktype_form"]
+        response = form.submit("_export")
+        data = response.content
+
+        url = reverse("admin:catalogi_catalogus_import_zaaktype", args=(catalogus.pk,))
+        response = self.app.get(url)
+
+        form = response.form
+        f = io.BytesIO(data)
+        f.name = "test.zip"
+        f.seek(0)
+        form["file"] = (
+            "test.zip",
+            f.read(),
+        )
+
+        zaaktype.datum_begin_geldigheid = datetime(2022, 1, 1).date()
+        zaaktype.datum_einde_geldigheid = datetime(2022, 12, 31).date()
+        zaaktype.save()
+
+        response = form.submit("_import_zaaktype").follow()
+        response = response.form.submit("_select")
+
+        self.assertEqual(response.status_code, 200)
+
+        error_text = (
+            _(
+                "Dit {} komt al voor binnen de catalogus en opgegeven geldigheidsperiode."
+            )
+            .format(informatieobjecttype._meta.verbose_name)
+            .title()
+        )
+        expected_html = f'<ul class="errorlist"><li>begin_geldigheid: overlap — {error_text}</li></ul>'
+
+        self.assertIn(
+            expected_html, response.text,
```

Comment:
There does not seem to be an easy way to test if the error appears properly. Might be easier if I use Django TestCase instead of Webtest

Reply:
If you want another way to search for error list I can think of two other options: 

1. You can use context from response both here and in usual Django TestCase:
``` iotype_forms = response.context["iotype_forms"] ```
And then assert iotype_forms.errors

2. In Webtest you can use `response.html` which is a Beautiful Soup object to scrape web page
It helps to look through html, so you can make sure that you have an error list rendered together with the correct iotype with somethink like this:
    ```
    iotype_tag = response.html.find(id="id_iotype-0-existing")
    errorlist_tag = iotype_tag.parent(class_="errorlist")
    ```
    Then you can assert that this tag exists and has the expected content

Reply:
I swear I tried iotype_forms.errors but it didn't work for me. Anyway, I changed it.